easylist_adult/adult_specific_hide.txt

```diff
@@ -1069,7 +1074,7 @@ akiba-online.com##[href^="https://filejoker.net/invite"]
 sweethentai.com##[href^="https://gadlt.nl/"]
 usasexguide.nl##[href^="https://instable-easher.com/"]
 adultcomixxx.com##[href^="https://join.hentaisex3d.com/"]
-ryuugames.com,yaoimangaonline.com##[href^="https://l.erodatalabs.com/s/"] > img
+yaoimangaonline.com##[href^="https://l.erodatalabs.com/s/"] > img
```

Comment:
I don't see a reason to remove this.

Reply:
But maybe then it is worth adding a rule that will remove this irritating pink color?

```adblock
ryuugames.com###wpfront-notification-bar:style(background: unset !important;)
```